benchmark/polyfills.js

```diff
@@ -0,0 +1,187 @@
+'use strict';
+
+/**
+ * Polyfills for older Node.js versions (0.8+)
+ * This file provides implementations for features not available in older Node versions
+ */
+
+// Polyfill for Array.from (ES6)
+if (!Array.from) {
```

Comment:
please use https://npmjs.com/array.from and `require('array.from/auto')` for this.